README.MD

```diff
@@ -99,6 +105,12 @@ influx_db:
     # Setup timeout or connection timeout (seconds) for your requests
     timeout:              0.0
     connect_timeout:      0.0
+            
+    # Set it to false to disable the event listener configuration
+    listener_enabled:     true
+
+    # Simple override for the default event listener class (constructor args and methods must match)
```

Comment:
@smatyas maybe, will be better to create an interface/abstract listener and force the user to implement it ?

Reply:
I wouldn't force the user to any of them. I think that when a user wants to write his own listener, it is because he needs his own, totally different services/parameters to be injected into his listener. And for those arbitrary needs we cannot really write a universal interface/abstract listener and build a correct definition for that in the extension class easily. Maybe he even wants to listen to totally different events with different priorities. These would be too much to configure IMHO, it's just easier to configure his own listener.

So I believe that the default, bundled implementation is perfect for a lot of cases, and if someone wants other things, we should just let him do whatever he wants in the standard ways of creating listeners, and be able to disable the bundled one. Maybe this `listener_class` is not a good idea either. It could be misleading, because if someone wants to include a custom service, he will hit the walls shortly. Probably a more detailed example of configuring a custom listener would be more helpful, and I should just remove this config option, and add more doc :)

What do you think?

Reply:
It is ok for me. Give me some time to do some testing.